src/cases/components/case-task/case-task.component.ts

```diff
@@ -62,6 +62,15 @@ export class CaseTaskComponent implements OnInit {
     if (!task.description) {
       return '';
     }
+    // Append task id as a querystring to the task description url
+    // The url format will be like the following markdown
+    // 'Click link to proceed to next step [test link next step](/cases/case-details/1547652071308205/trigger/editAppealAfterSubmit)'
+    const taskDescription = task.description;
```

Comment:
Can you move this logic to a util function and write some unit tests just for that?

Reply:
Please create a new src/cases/components/case-task/src/cases/components/case-task/case-task.util.ts

Reply:
Thank you @udaydenduluri33, I have now implemented the code as you suggested.